diff --git a/pyproject.toml b/pyproject.toml
index dd473b6..50eea42 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -13,7 +13,7 @@ requires-python = ">=3.10,<4"
 dependencies = [
   "opentelemetry-api>=1.38.0,<2",
   "opentelemetry-instrumentation>=0.59b0",
-  "opentelemetry-semantic-conventions-ai>=0.5.2,<0.6.0",
+  "opentelemetry-semantic-conventions-ai>=0.5.1,<0.6.0",
   "opentelemetry-semantic-conventions>=0.63b1",
 ]
 
